### PR TITLE
Chuwong/caretaker home

### PR DESCRIPTION
**Pull request by chuwong35122**
**Feature**
- Create UI for caretaker home screen and its function
- refactor many codes
- integrate with @bweam 's add elderly function
Screenshot_1648396068

### CODE REVIEW

File: src/components/organisms/UserCard.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+};
+
+const UserCard = ({ name, imageId, uid }: UserCardProps) => {
+  const navigation =
+    useNavigation<NativeStackNavigationProp<RootStackParamList>>();
+
+  const { t } = useTranslation();
+
+  function handlePressTakeCare() {
+    console.log('clicked');
```
Comment (pointmekin) [inline src/components/organisms/UserCard.tsx]: Don't forget console log UwU

File: src/screens/CaretakerHomeScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          <Divider />
+          <UpComingAlert />
+          {/* <ScrollView
+            horizontal
+            showsHorizontalScrollIndicator={false}
+            scrollEnabled={false}
+            width="100%" 
+  bgColor="red.100"> */}
+          <ElderlyInCareList />
+          {/* </ScrollView> */}
```
Comment (pointmekin) [inline src/screens/CaretakerHomeScreen.tsx]: Consider removing unused code

File: src/screens/caretakers/ConfirmConnectScreen.tsx [hunk 3]
```diff
@@ -9,65 +9,59 @@ import { RootStackParamList } from '../../navigation/types';
...
-      }).then(() => {
-        navigation.navigate('ConnectElderlyScreen');
-      })
-      .catch((err) => {
-        console.log({ err });
+    try {
+      const { data } = await client.post('/user/relationship', {
+        eid: info.uid,
+        cid: user?.uid
       });
```
Comment (pointmekin) [inline src/screens/caretakers/ConfirmConnectScreen.tsx:34]: here too

File: src/screens/caretakers/ConfirmConnectScreen.tsx [hunk 4]
```diff
@@ -9,65 +9,59 @@ import { RootStackParamList } from '../../navigation/types';
...
+      // const elderlyResult = data as Elderly;
+      navigation.navigate('ConnectElderlyScreen');
```
Comment (pointmekin) [inline src/screens/caretakers/ConfirmConnectScreen.tsx:36]: consider removing this

File: src/components/organisms/UserCard.tsx (hunk 1 above)
Comment (chuwong35122) [reply src/components/organisms/UserCard.tsx]: ok UwU

File: src/screens/caretakers/ConfirmConnectScreen.tsx (hunk 4 above)
Comment (chuwong35122) [reply src/screens/caretakers/ConfirmConnectScreen.tsx:36]: In case Beam wants to use the result

**Review by pointmekin — APPROVED** [review]

**Pull request merged**